doc/joss-paper/paper.md

```diff
@@ -0,0 +1,154 @@
+from the seeds using the colliding fronts methods (center), and finally
+evolving the isosurface through the level set equations (right).
+
+## Generating Centerlines from a Surface
+
+Centerlines are determined as the paths defined on Voronoi diagram sheets that
+minimize the integral of the radius of maximal inscribed spheres along the
+path, which is equivalent to finding the shortest paths in the radius metric.
+
+``vmtkcenterlines -ifile foo.vtp -ofile foo_centerlines.vtp``
+
+![centerline generation](centerlines.png)
+
+The input surface representation (left), a visualization of the internal subset
+of the voronoi diagram where each sheet represents a maximum inscribed sphere
+radius centered at some point in the surface (center), the centerline extracted
+from the voronoi diagram rendered as in it's position within the input surface
+(right).
+
+## Splitting a Surface from it's Centerlines
+
+Surface properties can be analyzed, and the surface can be split by analyzing
+the surface-centerline tube containment relationships.
+
+``vmtksurfacereader -ifile foo.vtp --pipe vmtkcenterlines --pipe
+vmtkbranchextractor --pipe vmtkbranchclipper -groupids 0 -insideout 1 -ofile
+foo_sp.vtp``
+
+![centerline splitting](splitting.png)
+
+Illustration of the centerline-surface tube containment relationships (left),
+visualization of the surface being split into independent groups based on the
+surfaces membership in a centerline tract/group (right).
+
+## Generating a Volumetric Mesh from a Surface
+
+Tetrahedral, mixed tetrahedral, and boundary layer meshes can be generated from
+a surface and it's centerlines. 
+
+``vmtksurfacereader -ifile foo.vtp --pipe vmtkcenterlines --pipe
+vmtkdistancetocenterlines -useradius 1 --pipe vmtkmeshgenerator
+-elementsizemode edgelengtharray -edgelengtharray DistanceToCenterlines
+-edgelengthfactor 0.3 -boundarylayer 1 -ofile foo.vtu``
+
+![ mesh generation](mesh.png)
+
+The input surface representation (left), a visualization of the internal mesh
+(center), a visualization of a boundary layer internal mesh (right).
+
+## Composable Scripting With PypeS
+
+``PypeS`` is the glue among vmtk scripts. It allows new scripts to be written
+easily and have a common interface, but, most of all, it allows single vmtk
+scripts to interact with each other, making ``VMTK`` modular and flexible.
+Pypes can be used from the command line, a python interpreter, or from the
+custom ``PypePad`` user interface
+
+![pypepad](pypepad.png)
+
+The VMTK PypePad user interface.
+
+# Acknowledgements & Funding
+
+The Vascular Modeling Toolkit is a community project which is open-source and
+free of charge. We would especially like to recognize significant
+infrastructure contributions from the team at Kitware Inc. (www.kitware.com).
+In addition we would like to acknowledge Orobix Srl. (www.orobix.com) for
+partially funding continued development efforts of ``VMTK``.
+
```

Comment:
It would be nice to acknowledge the main contributors listed in the source files, if there's room.

Reply:
Done!